### PR TITLE
new regulator controller class with xf as member variable. fixed get_…

### LINKED ISSUE #5: get_control
This cannot be correct. K should be acting on error, i.e. `x_current - x_final`, rather than just acting on the state. Caught the error when visualizing converged K and dare K with xf/goal state other than [0 0 0 0]. 

```py
    def get_control(self, x):
        '''
        '''
        return -self.K @ x
```

proposed fix:
```py
    def get_control(self, x, xf):
        '''
        '''
        return -self.K @ (x-xf)
```

And need to add
```py
            self.U.append(controller.get_control(self.X[-1], self.xf))
```
in https://github.com/jsalfity/sensitivity_games/blob/main/sensitivity_games/trajectory.py#L25.

@dfridovi -- Question is how to elegantly introduce xf into the abstraction architecture? Is that how you would do it?

https://github.com/jsalfity/sensitivity_games/blob/cfa75bf6eb892ef098bb81e08ec1e981753d2018/sensitivity_games/linear_feedback_controller.py#L39-L42

### PR DESCRIPTION
**Pull request by jsalfity**
…control in linear feedback controller

### CODE REVIEW

**Comment by dfridovi** [comment]
About to review. Pro tip - add short description of feature as PR name, then longer form description below. Also make sure to mention issue so GitHub links it.

File: sensitivity_games/controller.py [hunk 1]
```diff
@@ -13,6 +13,16 @@ def get_control(self, x):
         raise NotImplementedError('Not Implemented!')
 
 
+class RegulatorController(Controller):
```
Comment (dfridovi) [inline sensitivity_games/controller.py]: nit: this and next class should be in separate files.... but not a huge deal of course

File: sensitivity_games/point_mass.py [hunk 2]
```diff
@@ -89,13 +89,13 @@ def modify_B(self):
...
 
-    def visualize(self, X, U, goal, block):
+    def visualize(self, X, U, xf, block):
         '''
         '''
-        xmax = max([x[0].detach() for x in X])
-        xmin = min([x[0].detach() for x in X])
-        ymax = max([x[2].detach() for x in X])
-        ymin = min([x[2].detach() for x in X])
+        xmax = max(max([x[0].detach() for x in X]), xf[0])
```
Comment (dfridovi) [inline sensitivity_games/point_mass.py:95]: wait don't you want to look at error somewhere? I'm confused why there are two maxes here

**Review by dfridovi — APPROVED** [review]
A couple questions, but generally looks good to me.

File: sensitivity_games/point_mass.py (hunk 2 above)
Comment (jsalfity) [reply sensitivity_games/point_mass.py:95]: this is just used for plotting, so i can set the axes correctly.

File: sensitivity_games/controller.py (hunk 1 above)
Comment (jsalfity) [reply sensitivity_games/controller.py]: ah, right, i'll fix.

**Pull request merged**